addons/html_editor/static/tests/history.test.js

```diff
@@ -438,3 +439,44 @@ describe("shortcut", () => {
         ]).toVerifySteps();
     });
 });
+
+describe("destroy", () => {
+    test("Mutations are not observed after history plugin is destroyed", async () => {
```

Comment:
Not sure we should keep this test, it essentially ensure that the observer is disconnected during cleanup.

But this is a scenario of what could go wrong if the observer was not disconnected